TestCases/wallfunctions/flatplate/turb_SA_flatplate.cfg

```
@@ -0,0 +1,228 @@
+% SU2 configuration file                                                       %
+% Case description: Turbulent flow over flat plate with zero pressure gradient %
+% Author: Thomas D. Economon                                                   %
+% Institution: Stanford University                                             %
+% Date: 2011.11.10                                                             %
+% File Version 7.1.1 "Blackbird"                                                %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= RANS
+%
+% If Navier-Stokes, kind of turbulent model (NONE, SA)
+KIND_TURB_MODEL= SA
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
+MATH_PROBLEM= DIRECT
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO 
+%
+
+% ----------- COMPRESSIBLE AND INCOMPRESSIBLE FREE-STREAM DEFINITION ----------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 0.2
+%
+% Angle of attack (degrees)
+AOA= 0.0
+%
+% Side-slip angle (degrees)
+SIDESLIP_ANGLE= 0.0
+%
+% Free-stream temperature (288.15 K by default)
+FREESTREAM_TEMPERATURE= 300.0
+%
+% Reynolds number (non-dimensional, based on the free-stream values)
+REYNOLDS_NUMBER= 5000000.0
+%
+% Reynolds length (in meters)
+REYNOLDS_LENGTH= 1.0
+
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+%
+% Reference origin for moment computation
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+%
+% Reference length for pitching, rolling, and yawing non-dimensional moment
+REF_LENGTH= 1.0
+%
+% Reference area for force coefficients (0 implies automatic calculation)
+REF_AREA= 2.0
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+%
+% Navier-Stokes wall boundary marker(s) (NONE = no marker)
+MARKER_HEATFLUX= ( wall, 0.0 )
+
+%
+% Inlet boundary marker(s) (NONE = no marker) 
+% Format: ( inlet marker, total temperature, total pressure, flow_direction_x, 
+%           flow_direction_y, flow_direction_z, ... )
+MARKER_INLET= ( inlet, 302.4, 118309.784, 1.0, 0.0, 0.0 )
+%
+% Outlet boundary marker(s) (NONE = no marker)
+% Format: ( outlet marker, back pressure, ... )
+MARKER_OUTLET= ( outlet, 115056.0, farfield, 115056.0 )
+%
+% Symmetry boundary marker(s) (NONE = no marker)
+MARKER_SYM= ( symmetry )
+%
+% Marker(s) of the surface to be plotted or designed
+MARKER_PLOTTING= ( wall )
+%
+% Marker(s) of the surface where the functional (Cd, Cl, etc.) will be evaluated
+MARKER_MONITORING= ( wall )
```

```suggestion
MARKER_HEATFLUX= ( wall, 0.0 )
MARKER_INLET= ( inlet, 302.4, 118309.784, 1.0, 0.0, 0.0 )
MARKER_OUTLET= ( outlet, 115056.0, farfield, 115056.0 )
MARKER_SYM= ( symmetry )
MARKER_PLOTTING= ( wall )
MARKER_MONITORING= ( wall )
```
etc., description is in the template, let's not duplicate things

---

I'd rather keep the comments in general, because I think it makes it easier to read the file, especially when you are going to use these testcases as a starting point for your own setup. Would be good to remove all lines that are ignored in the setup though.

---

I changed my mind about keeping the comments. I cleaned up the testcases and I have added an incomp SA, incomp SST, comp SA and comp SST testcase.